app/experimenter/static/rapid/package.json

```diff
@@ -46,7 +46,7 @@
     "build": "yarn clean; webpack --mode production --progress",
     "clean": "rimraf ../assets/rapid",
     "lint": "yarn lint:eslint; yarn lint:tsc",
-    "lint:eslint": "eslint --no-error-on-unmatched-pattern --color --max-warnings 0 ./**/*.js ./**/*.ts ./**/*.tsx",
+    "lint:eslint": "eslint --no-error-on-unmatched-pattern --color --max-warnings 0 .",
```

Comment:
i know it's more verbose but i think instead of linting js files in the root which will always be config files we should do:
`./*.ts ./*.tsx ./**/*.ts ./**/*.tsx ./**/*.js`

this way we still catch any files we might care about in the root like `index.tsx` and ignore the config files.

Reply:
Sure, or what about if we just did an .eslintignore for the jest/webpack/eslint config files in the root?

**PROS:**
- easier to understand config.
- would catch any future non-config *.js files in the root folder.

**CONS:**
- one more config file.

Probably a bit moot, as we should probably tweak the ESLint config eventually and move it higher up the filesystem and just override configs using https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns vs cascading .eslintrc files throughout the directory structure.

Reply:
```sh
DEBUG=eslint:cli-engine yarn lint:eslint
yarn run v1.22.4

$ eslint --no-error-on-unmatched-pattern --color --max-warnings 0 ./*.ts ./*.tsx ./**/*.ts ./**/*.tsx ./**/*.js
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/experimenter/app/experimenter/static/rapid/index.tsx +0ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/experimenter/app/experimenter/static/rapid/components/App.tsx +270ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/experimenter/app/experimenter/static/rapid/__tests__/app.test.js +51ms
  eslint:cli-engine Linting complete in: 1098ms +175ms
✨  Done in 1.44s.
```

Reply:
ah yes, I was trying to find an eslint ignore option but I guess I didnt dig hard enough. that wfm